letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -285,7 +287,15 @@ DeterminePythonVersion() {
     fi
   fi
 
-  PYVER=`"$LE_PYTHON" -V 2>&1 | cut -d" " -f 2 | cut -d. -f1,2 | sed 's/\.//'`
+  if [ "$USE_PYTHON_3" = 1 ]; then
+    MIN_PYVER=$MIN_PYVER3
```

Comment:
These variables are used elsewhere in certbot-auto (see [here](https://github.com/certbot/certbot/blob/2883ca839e000912a7e524c4087bb9f66f8affef/certbot-auto#L1618) and [here](https://github.com/certbot/certbot/blob/2883ca839e000912a7e524c4087bb9f66f8affef/certbot-auto#L1789)). Unfortunately, this makes things more complex, but I don't immediately see a better way around that.

One problem here though is if `NOCRASH` is specified as it is [here](https://github.com/certbot/certbot/blob/2883ca839e000912a7e524c4087bb9f66f8affef/certbot-auto#L1615-L1618), these variables may not be defined before they are used.

To help resolve this, I propose:

1. Merge this conditional with the earlier one in this function and set these variables before the `for` loop.
2. Update the comment above this function to say that these variables are also set here.

Reply:
Done.